### ISSUE TITLE
Several Learn More controls may appear on the page at one time, but there is no way to differentiate one from another

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1355]
From Level Access:

Ensure custom controls provide proper textual name, role, and state information.

[Issue]
Several Learn More controls may appear on the page at one time, but there is no way to differentiate one from another.

[Impact]
Assistive technology users may have difficulty when trying to determine which control is associated with a particular benefit.

Lack of Unique Labels for Controls

[Expected Behavior]
Ensure that similar custom controls have an accessible name that describes it relationship to the content.

**Comment by smcmurtry** [comment]
This is no longer an issue because the benefit title appears in the learn more button
